inc/customizer/colors.php

```diff
@@ -325,7 +332,11 @@ public function __construct() {
 						a.fl-button, .content-area .fl-builder-content a.fl-button, .content-area .fl-builder-content a.fl-button:visited, .content-area .fl-builder-content a.fl-button *, .content-area .fl-builder-content a.fl-button:visited *, a.fl-button:hover, a.fl-button:active, a.fl-button:focus, a.fl-button:visited, a.fl-button:visited:hover, a.fl-button:visited:active, a.fl-button:visited:focus,
 						input[type="button"], input[type="button"]:hover, input[type="button"]:active, input[type="button"]:focus,
 						input[type="reset"], input[type="reset"]:hover, input[type="reset"]:active, input[type="reset"]:focus,
-						input[type="submit"], input[type="submit"]:hover, input[type="submit"]:active, input[type="submit"]:focus' => array(
+						input[type="submit"], input[type="submit"]:hover, input[type="submit"]:active, input[type="submit"]:focus,
+						.woocommerce a.button,  .woocommerce a.button:hover,
```

Comment:
@EvanHerman When we are targeting the `color` of an `<a>` tag, we have to be sure to include selectors for `:visited` and `:visited:hover` otherwise the original color scheme can sometimes pop up and surprise you, depending on the child theme.